nvdaHelper/vbufBackends/adobeAcrobat/adobeAcrobat.cpp

```diff
@@ -692,11 +694,14 @@ AdobeAcrobatVBufStorage_controlFieldNode_t* AdobeAcrobatVBufBackend_t::fillVBuf(
 		}
 
 		// Hereafter, tempNode is the text node (if any).
-		tempNode = renderText(buffer, parentNode, previousNode, domNode, domElement, useNameAsContent, parentNode->language, textFlags, pageNum);
-		if (tempNode) {
-			// There was text.
-			previousNode = tempNode;
-		}
+		if (domNode) {

```

Comment:
No; pageNum can be NULL if there is no page number and `renderText` checks for this.